### DISCUSSION TITLE
Support for arm64 docker images?

### DISCUSSION THREAD

**Discussion by yatharthranjan** [RADAR-base/RADAR-Docker#257] [Ideas]
Just a question, should we start supporting arm64 arch in docker images? This is getting more popular now with M1 MacBooks and also some server offerings (like AWS Graviton2). Another advantage is being able to run components on IoT devices like raspberry pi (the latest raspbian now supports arm64).
Not sure how feasible this will be though, since some base images, libs, etc required for building might now be available on arm64 but still worth investigating.
Any thoughts @afolarin @blootsvoets?

**Comment by blootsvoets** [comment]
All of the GitHub actions where the docker images are built as part of the release have an arm64 platform tag. Indeed, that could be expanded to all images components. Currently supported:

- kafka-connect-transform-keyvalue
- radar-gateway
- radar-output-restructure
- radar-rest-source-auth
- radar-schemas

**Reply by yatharthranjan** [reply]
Also, the official confluent docker images don't support arm64, https://github.com/confluentinc/common-docker/issues/117, so these would need to be built too.

**Comment by afolarin** [comment]
Makes sense to support this, as you say both server-side, Apple and IoT will likely drive this requirement in the near term.
